test/integration/auction.test.js

```diff
@@ -14,7 +14,7 @@ const { createWallet } = require("../../models/wallets");
 const userData = require("../fixtures/user/user")();
 const { auctionData, auctionKeys, auctionWithIdKeys } = require("../fixtures/auctions/auctions");
 const { initial_price: initialPrice, item_type: itemType, end_time: endTime, quantity } = auctionData;
-const currenciesData = require("../fixtures/currencies/currencies");
+const currenciesData = require("../fixtures/currencies/currencies")[0];
```

Comment:
You can destructure the currenciesData on the next line it will be more readable. 

like this: 
`const currencyData = currenciesData[0];` or `[const currencyData] = currenciesData`

Reply:
Made the changes, is that ok?